### PR TITLE
Fix translation error (review requested)

### PR DESCRIPTION
**Pull request by HelloSeaNation**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
will ask for review

File: docs/CONTRIBUTING-zh.md [hunk 1]
```diff
@@ -175,120 +175,116 @@
...
+##### 网址
 
-- We don't permit shortened URLs.
-- Tracking codes must be removed from the URL.
-- International URLs should be escaped. Browser bars typically render these to Unicode, but use copy and paste, please.
-- Secure (`https`) URLs are always preferred over non-secure (`http`) urls where HTTPS has been implemented.
-- We don't like URLs that point to webpages that don't host the listed resource, but instead point elsewhere.
+- 我们不允许使用短链。
+- 必须从网址中删除追踪代码。
+- 国际网址应进行转义。浏览器列通常会将这些内容呈现为 Unicode，但请使用复制和贴上。
```
Comment (kaichii) [inline docs/CONTRIBUTING-zh.md:205]: Changing it to「网址应该进行转义。浏览器地址栏的网址通常已被转义，可以去那里拷贝。」 might make it easier to understand.

File: docs/CONTRIBUTING-zh.md [hunk 2]
```diff
@@ -175,120 +175,116 @@
...
-- websites (except for those that host LOTS of items that we list).
-- videos that aren't courses or screencasts.
-- book chapters
-- teaser samples from books
-- IRC or Telegram channels
-- Slacks or mailing lists
+##### 书籍与其他东西
 
-Our competitive programming lists are not as strict about these exclusions. The scope of the repo is determined by the community; if you want to suggest a change or addition to the scope, please use an issue to make the suggestion.
+我们对书本性并不那么挑剔。以下是一些表示资源是一本书的属性：
```
Comment (kaichii) [inline docs/CONTRIBUTING-zh.md:254]: 我们对于书本的性质并不那么挑剔。以下是一些表明资源是一本书的特征：

**Comment by eshellman** [comment]
thanks @kaichii for review, @HelloSeaNation  please address comments

File: docs/CONTRIBUTING-zh.md (hunk 1 above)
Comment (HelloSeaNation) [reply docs/CONTRIBUTING-zh.md:205]: Thanks for the suggestion, changes is made. (Done)

File: docs/CONTRIBUTING-zh.md (hunk 2 above)
Comment (HelloSeaNation) [reply docs/CONTRIBUTING-zh.md:254]: Done. and Thanks!

**Comment by HelloSeaNation** [comment]
I have corrected the sugestions from all the reviewer

**Pull request merged**